components/pages/index/HomePage.js

```diff
@@ -41,7 +47,7 @@ const HomePage = () => {
               speed.
             </TagLine>
             {user ? (
-              <button type="button" style={{ width: '200px', height: '60px' }}>
+              <button type="button" style={{ width: '200px', height: '60px' }} onClick={handleClick}>
```

Comment:
It should be better to use <Link> for changing route. You can use `ButtonedLink` component with custom styles
https://nextjs.org/docs/api-reference/next/link

```
import Link from 'next/link';
import ButtonedLink from 'components/shared/atoms/ButtonedLink';

<Link href={GAME} passHref>
  <ButtonedLink>начать новую игру</ButtonedLink>
</Link>

```